src/components/SplashPage/styles/SplashPage.scss

```diff
@@ -0,0 +1,151 @@
+@import '../../../static/css/variables.scss';
```

Comment:
The tabbing in this file seems to be inconsistent, but I think we can come back for more formatting cleanup in a latter PR. There's a lot we need to change in the codebase after we define some styling rules we want to follow and get on board with some formatters.